runelite-client/src/main/java/net/runelite/client/util/Text.java

```diff
@@ -62,4 +70,116 @@ else if (!inTag)
 		return builder.toString();
 	}
 
+	/**
+	 * Creates a color tag from the given `color`.
+	 *
+	 * @param color The Color to create a tag from.
+	 * @return      A string of the color tag for the given `color`.
+	 */
+	public static String colorTag(Color color)
```

Comment:
Since all color formatting should go trough this class, this can probably be private?

Reply:
It can't be made private as it is used [here](https://github.com/runelite/runelite/pull/4072/files#diff-bf265f5ad83eea9b469bdd43046c6ce2R137) and [here](https://github.com/runelite/runelite/pull/4072/files#diff-726aed9d5db03fc1de5694de6970f8b2R90).